HowTo install Linux (Ubuntu) tools for C++ programming with Eclipse IDE/readme.md

```diff
@@ -25,17 +25,17 @@ Afterwards, the check should look like this:
 
 ![eclipse package](images/eclipse_package.png)
 
-or use wget to download **2019.3 R** version (risk of being changed):
+or use wget to download **2020.0 R** version (risk of being changed):
```

Comment:
Actual Eclipse version is **2020‑03 R**. You should correct this steps description.